### ISSUE TITLE
simBuyseTest time to event data: rates vs. scales

### ISSUE THREAD

**Issue by graemeleehickey** [bozenne/BuyseTest#4]
When running

```r
#### only TTE endpoints ####
args <- list(rates.T = c(3:5/10), rates.Censoring.T = rep(1,3))
simBuyseTest(100, argsBin = NULL, argsCont = NULL, argsTTE = args)
```

I think the rates are actually **scales**.  This can be see by graphically displaying the survival curve:
```r
plot(survfit(Surv(eventtime3, status3) ~ Treatment, data = dat))
x <- seq(0, 2.5, len = 1001)
y1 <- exp(-0.5 * x)
lines(x, y1, col = 2)

y2 <- exp(-1/0.5 * x)
lines(x, y2, col = 3)
```

The red curve (rate = 0.5) does not match the simulated the data. However, the green curve (rate = 2, scale = 0.5) does.

**Comment by bozenne** [comment]
Thanks -sorry for the late answer (I am making some other changes in the package) I let you know when I fix that.

**Comment by bozenne** [comment]
I think this is an issue with the function coxExponential.lvm from the lava package that I use to generate the data. I have asked the maintainer to rename the argument.

**Comment by bozenne** [comment]
Ok so that lava function won't be changed. So I have modified simBuyseTest such that it should match the usual parametrisation. Available on version 1.8.1 and should be on its way to CRAN very soon (say within 2 weeks).
